tests/test_compare.py

```diff
@@ -103,3 +104,58 @@ def test_uncertaintychange(self):
                 if not accumulate_changes:
                     modelcp = cp(model)
         assert True, "Failure in uncertainty changing test"
+
+    def test_missing_uncertainties(self):
+        # Removes uncertainties from both models and attempts to use compare.
+
+        par_base1 = """
+            PSR J1234+5612
+            RAJ 14:34:01.00 
+            DECJ 56:14:00.00
+            F0 1 
+            PEPOCH 57000
+            DM 10 
+            DMEPOCH 57000
+            DM1 2
+            DMX     0.0
+            DMX_0001   1.0 
+            DMXR1_0001     58000.0
+            DMXR2_0001     58000.0
+            """
+
+        par_base2 = """
```

Comment:
Is this actually any different from par_base1? If so it might be good to have a comment to indicate how. If not why not just reuse par_base1?

Reply:
nope, good point. i removed par_base2